terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1 +1,62 @@
-# placeholder file
\ No newline at end of file
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+
+# TODO Check not required (not there in OE Web Staging config)
```

Comment:
A number of these TODOs in this file, if people could weigh in, would be good. Or we can have a call.

Reply:
No they're not needed

Reply:
👍 thanks.

Reply:
Now removed. Resolving this comment.